src/backend_stub/index.js

```diff
@@ -18,6 +18,13 @@ module.exports = {
 		var app = express(http.createServer());
 		var appServer;
 		
+		app.use(config.staticBaseUrl, serveStatic(config.staticPath));
```

Comment:
Ah, wait. This ```_stub.js``` file is a "mock" for C++ code, right?

If yes, then it **suddenly** makes a lot more sense to me!